gitclone/checkout_method_pr_auto_merge_branch.go

```diff
@@ -40,8 +40,15 @@ func (c checkoutPRMergeRef) do(gitCmd git.Git, fetchOpts fetchOptions, fallback
 	// https://git-scm.com/book/en/v2/Git-Internals-The-Refspec
 	refSpec := fmt.Sprintf("%s:%s", c.remoteMergeRef(), c.localMergeRef())
 
+	// delete remote branch ref
```

Comment:
Can you add a comment explaining why we need this? It's not trivial and no one will remember in a year

Reply:
Added comment.